eos-chrome-plugin-update-upstream

```diff
@@ -21,6 +21,21 @@ DEST_DIR=/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater
 WGET_COMMAND="wget -c --tries=5"
 DAYS_CHECK=7
 
+# Data needed for the Flash plugin, from Adobe
+FLASH_FALLBACK_VERSION="23.0.0.185"
```

Comment:
I don't think we can do that right now, but if we follow my proposal from https://github.com/endlessm/eos-chrome-plugin-updater/pull/18/commits/4c9e3eb784bb9dd84a6066ea9ffd068bfd2c38fd#r84042446 (see the end of the reply) where we'd have a Jenkins job updating the LKVG file periodically, then we could update this package so that this fallback information gets picked from that LKGV at the time of building the package.